app/models/driver.rb

```diff
@@ -7,5 +7,5 @@ class Driver < ApplicationRecord
   validates :mobile_number, presence: true, phone: true
   validates :email_address, presence: true, email: true
   validates :age, presence: true, numericality: { greater_than_or_equal_to: 18, less_than_or_equal_to: 74 }
-  validates :status, presence: true, inclusion: { in: [ true, false ] }
+  validates :status, inclusion: { in: [ true, false ] }
```

Comment:
As it is boolean field, you can add `default: false` to the db migration

Reply:
> As it is boolean field, you can add `default: false` to the db migration

Does that mean that all boolean fields will require `default: false` to be added?

Also, will I have to run the migration again after this change?

Reply:
If you forbid `null` value it is better to set some default for the boolean fields.
The default value depends on the application requirements.

Reply:
it would have to be a new migration where we change the default for the colun
we cant update existing migrations that have already run